astro/src/styles/components/SearchForm.css

```diff
@@ -0,0 +1,4 @@
+
+.light-gray-border {
```

Comment:
@iam-benyamin sorry to be annoying. whenever possible, try to use a color that is provided by tailwind. If it is not provided, generate a new color serie (https://uicolors.app/create) and add it to the tailwind config file, so you can use it in any class. 
IE: you create a new color called `mygrey` and you can use classes like `text-mygrey-100` or in this case `border-mygrey`